src/infix-abstract.ts

```diff
@@ -0,0 +1,71 @@
+
+const SUBSCRIPT_DIGITS = new Map(Object.entries({
+  "0": "₀",
+  "1": "₁",
+  "2": "₂",
+  "3": "₃",
+  "4": "₄",
+  "5": "₅",
+  "6": "₆",
+  "7": "₇",
+  "8": "₈",
+  "9": "₉",
+  "-": "₋"
+}));
+
+
+import { Notation } from "./notation";
+import Decimal from "break_infinity.js";
+
+export abstract class AbstractInfixNotation extends Notation {
+  public get name(): string {
+    return "Abstract Infix";
+  }
+
+  private groupDigits = 3
```

Comment:
Codacy found an issue: [expected member-variable-declaration: 'groupDigits' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)